code/game/gamemodes/gang/gang_things.dm

```diff
@@ -40,10 +41,13 @@
 		to_chat(user, policy)
 	team_to_use.add_member(user.mind)
 	for(var/threads in team_to_use.free_clothes)
-		new threads(get_turf(user))
+		var/obj/O = new threads(get_turf(user))
+		O.armor = list("melee" = 7.5, "bullet" = 7.5, "laser" = 7.5, "energy" = 7.5, "bomb" = 7.5, "bio" = 7.5, "rad" = 7.5, "fire" = 7.5, "acid" = 7.5, "wound" = 7.5)
+	for(var/threads in team_to_use.current_theme.bonus_items)
+		var/obj/O = new threads(get_turf(user))
+		O.armor = list("melee" = 7.5, "bullet" = 7.5, "laser" = 7.5, "energy" = 7.5, "bomb" = 7.5, "bio" = 7.5, "rad" = 7.5, "fire" = 7.5, "acid" = 7.5, "wound" = 7.5)
```

Comment:
This should go into the object's definition.

Reply:
No, gang uniforms include common clothing pieces, this will not be happening.